docs/tasks.md

```diff
@@ -234,9 +232,7 @@ After you have generated the `ClusterAdmissionPolicy` and applied it to your Kub
       mutating: false
     ```
   
-    > TIP: By default, the `name` value is set to `generated-policy`. You might want to edit it before you deploy the `ClusterAdmissionPolicy`. 
-    >
-    > NOTE: To avoid confusion, the value above has been set to `privileged-pods`.
+    > TIP: By default, the `name` value is set to `generated-policy`. You might want to edit it before you deploy the `ClusterAdmissionPolicy`. The name in the immediately previous example has been set to `privileged-pods`.
```

Comment:
Maybe we can replace this kind of `> TIP` by [docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) as we do above in the page.

Reply:
Admonitions are great and stand out well. The tip is a bit less important so it's probably ok to not have it standout quite as much. Both mechanisms should be used sparingly otherwise a reader stops paying attention to them. But, yeah, we have two different styles of 'admonitions' which is not great from a consistency point of view. Another iteration perhaps.

Reply:
Hmmm, I think consistency is the best policy,so hold on a few minutes. :-)